server/src/main.rs

```diff
@@ -260,6 +260,11 @@ fn main() {
                         .long("--enable-mqtt-endpoint")
                         .help("enable mqtt endpoint"),
                 )
+                .arg(
+                    Arg::with_name("enable-mqtt-integration")
+                        .long("--enable-integration-endpoint")
```

Comment:
I think name and arg should both be '--enable-mqtt-integration' should be the same to be consistent with the other options. Also I imagine websocket would be '--enable-websocket-integration'.